frontend/webclient/script/handlers.js

```diff
@@ -119,6 +119,36 @@ $handlers.handle_postpro_result = function(msgobj) {
     $globals.professor_scope.$apply();
 };
 
+$handlers.handle_annotated_file = function(msgobj) {
+
+    $globals.professor_scope.annotated_files = [];
+
+    var files = msgobj.files;
+
+    for (var i = 0; i < files.length; i++) {
+			var file = {};
+			file.filename = files[i].filename;
+			var data_list = [];
+
+			var data_list_length = files[i].data.length;
+			for (var j = 0; j < data_list_length; j++) {
+				var data_entry = {};
+				data_entry.no = files[i].data[j].no;
+				data_entry.content = files[i].data[j].content;
+				data_entry.annotation = files[i].data[j].annotation;
+				//console.log('data pushed: ' + data_entry.no + ' ' + data_entry.content);
```

Comment:
This line could go away. Also, isn't the indentation of this file a bit weird? :') I realized only now!